bindata/network/ovn-kubernetes/managed/ovnkube-master.yaml

```diff
@@ -407,8 +407,12 @@ spec:
                 if [[ "${pod_dns_name}" == "${CLUSTER_INITIATOR_IP}" ]]; then
                   echo "$(date -Iseconds) - nbdb - postStart - waiting for master to be selected"
 
+                  listen_dual_stack=""
+                  if [[ "${K8S_POD_IP}" == *":"* ]]; then
+                    listen_dual_stack=":[::]"
```

Comment:
Hey @kyrtapz when you says `listen_dual_stack` it's also supporting single stack IPv6?